internal/gengapic/testdata/foo_client_init.want

```diff
@@ -1,35 +1,52 @@
+// internalFooClient is an interface that defines the methods availaible from Awesome Foo API.
+type internalFooClient interface {
+	Close () error
+	Zip(context.Context, *mypackagepb.Bar, ...gax.CallOption) (*mypackagepb.Foo, error)
+}
+
 // FooClient is a client for interacting with Awesome Foo API.
-//
 // Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.
 type FooClient struct {
+	// The internal transport-dependent client.
+	internalFooClient
+
+	// The call options for this service.
+	CallOptions *FooCallOptions
+
+}
+
+// fooGrpcClient is a client for interacting with Awesome Foo API over gRPC transport.
+// It satisfies the internalFooClient interface.
+//
+// Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.
+type fooGrpcClient struct {
```

Comment:
I'm surprised this type isn't exported, but the one in `empty_client_init.want` is?

Reply:
It's an artifact of the empty client test. I can fix it if need be, but it's purely a testing issue.